app/services/sufia/user_stat_importer.rb

```diff
@@ -46,81 +54,93 @@ def sorted_users
 
     private
 
-      def delay
-        sleep @delay_secs
-      end
+    def delay
+      sleep @delay_secs
+    end
 
-      def rescue_and_retry(fail_message)
-        retry_count = 0
-        begin
-          return yield
-        rescue StandardError => e
-          retry_count += 1
-          if retry_count < @number_of_retries
-            delay
-            retry
-          else
-            log_message fail_message
-            log_message "Last exception #{e}"
-          end
+    def rescue_and_retry(fail_message)
+      retry_count = 0
+      begin
+        return yield
+      rescue StandardError => e
+        retry_count += 1
+        if retry_count < @number_of_retries
+          delay
+          retry
+        else
+          log_message fail_message
```

Comment:
@jcoyne: I believe all that @gordonleacock has changed here is the indentation. We should probably handle this in a separate PR.